apps/web/pages/contact.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <NuxtLayout name="default">
+    <!-- TODO: remove this code when the re-captcha/back-end call is working -->
+    <div v-if="true" class="h-[500px] flex items-center justify-center">
+      <h1>{{ $t('coming-soon') }}</h1>
+    </div>
+    <div v-else class="md:max-w-[677px] mx-auto px-4 pt-4 pb-20 md:px-0 md:mt-4">
+      <h1 class="font-bold mb-10 typography-headline-3 md:typography-headline-2">
+        {{ $t('contact.contact') }}
+      </h1>
+      <p class="mb-10">{{ $t('contact.contactShopMessage') }}</p>
+
+      <form data-testid="contact-form" class="flex flex-col rounded-md gap-4" @submit.prevent="onSubmit">
+        <label>
+          <UiFormLabel class="mb-1">{{ $t('contact.form.nameLabel') }} *</UiFormLabel>
+          <SfInput name="name" type="text" v-model="contact.name" required />
+        </label>
+        <label>
+          <UiFormLabel class="mb-1">{{ $t('contact.form.emailLabel') }} *</UiFormLabel>
+
+          <SfInput name="email" type="email" autocomplete="email" v-model="contact.email" required>
+            <template #prefix>
+              <SfIconEmail />
+            </template>
+          </SfInput>
+        </label>
+        <label>
+          <UiFormLabel class="mb-1">{{ $t('contact.form.subjectLabel') }}</UiFormLabel>
+          <SfInput name="subject" type="text" v-model="contact.subject" />
+        </label>
+        <label>
+          <UiFormLabel class="mb-1">{{ $t('contact.form.order-id') }}</UiFormLabel>
+          <SfInput name="order-id" type="text" v-model="contact.orderId" />
+        </label>
+        <label>
+          <UiFormLabel class="mb-1">{{ $t('contact.form.message') }} *</UiFormLabel>
+          <SfTextarea
+            :placeholder="$t('contact.form.message-placeholder')"
+            class="w-full"
+            name="message"
+            v-model="contact.message"
+            required
+          />
+        </label>
+
+        <div class="flex items-center">
+          <SfCheckbox
+            :invalid="showPrivacyError"
+            @change="showPrivacyError = false"
+            id="terms"
+            v-model="privacyPolicy"
+            value="value"
+            class="peer"
+            required
+          />
+          <label class="ml-3 text-base text-neutral-900 cursor-pointer peer-disabled:text-disabled-900" for="terms">
+            <i18n-t keypath="contact.privacyPolicy">
+              <template #privacyPolicy>
+                <SfLink
+                  href="/PrivacyPolicy"
```

Comment:
this should use localepath right?

Reply:
also we might want to start using the /apps/web/utils/paths.ts file for the static routes?
This could also be its own task :)

Reply:
locale path modified, added paths for privacy policy. Ur right, might be a good idea to create a task for all of them. Nice catch, thanks!